src/Components/EventDetailsTile/EventDetailsTile.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Button } from "react-bootstrap";
+import styles from "./EventDetailsTile.module.scss";
+const EventDetailsTile = ({
+  background,
+  buttonText,
+  buttonColor,
```

Comment:
No need for button color (warning in all cases)

Reply:
prop to have customization in case kisi event ka card color yellow themed